app/controllers/games_controller.rb

```diff
@@ -4,10 +4,15 @@ def new
   end
 
   def index
-    @games = Game.all
+    # Queries 20 games
+    @games = Game.open_seats
   end
 
   def create
+    # Note that the creator of the game automatically
+    # becomes the black_player. Due to some issues in
+    # querying, for games index, the model only checks
+    # for games where the white_player_id is nil.
     @game = Game.create(game_params)
     @game.update_attribute(game_params)
```

Comment:
update_attributes isn't needed. Since those params get set by the previous line.